README.md

```diff
@@ -1,208 +1,256 @@
 
-<img src="assets/images/step2.png"/>
+For this purpose you need to:
+-	Add your Apple Push Notification service SSL certificate in `.p12` format
+-	Add server url and port. For sandbox it is `gateway.sandbox.push.apple.com:2195` and `gateway.push.apple.com:2195` for production.
+-	Add feedback url and port. For sandbox it is `feedback.sandbox.push.apple.com:2196` and `feedback.push.apple.com:2196` for production.
 
-### Step 3 - Select Devices
-Select the devices that you want to deploy your app to and upload splash screen images for the devices you have selected. 
+## Setting up Google Cloud Messaging Server
+
+In order to send google push notifications from this module you need to have set up a google account with google cloud messaging enabled.
+To do so follow these steps to get registered for Google cloud messaging and enter the details into the Mendix screens.
 
-Press the button publish to appstore and you will be asked whether you want to build in the cloud or do it yourself. Choose do it yourself and press the Download Phonegap Build Package.
+### Step 1 - Login to developers console
+
+Open up the Google [developers console] (https://console.developers.google.com) and login with your Google id.
+<img src="assets/images/gcm-step1.PNG"/>
 
+### Step 2 - Create project
 
-<img src="assets/images/step3.png"/>
+Click new project and fill in the project name and project ID for your application. Then click create. Once created you will see a project number at the top of the screen next to the project ID. Take note of this ID because you will need it later on for our sender ID.
 
-### Step 4 - Download phonegap.zip
+<img src="assets/images/gcm-step2.png"/>
 
-Once the phonegap.zip is downloaded, unzip it into a folder and open up the config.xml. You will need to edit the config.xml so that you can include additional phonegap plugins. The plugin we will need to include is the phonegap [push plugin](https://github.com/phonegap-build/PushPlugin).
+### Step 3 - Enable Google Cloud Messaging
 
-The code you will need to include is:
-`<gap:plugin name="com.phonegap.plugins.pushplugin" version="2.4.0" />`
+Once created, click the link to the Google Cloud Messaging API and click the Enable button.
+<img src="assets/images/gcm-step3.png"/>
 
-<img src="assets/images/step4.png"/>
+### Step 4 - Adding credentials
 
-### Step 5 - Configure xml
-Once you have edited the config.xml you should have everything necessary for your application to work. You will now need to zip up your files and upload the zipped file to [phonegap build](https://build.phonegap.com).
+Click on the menu option credentials, located on the left hand side under the API Manager section.
+<img src="assets/images/gcm-step4.png"/>
 
-<img src="assets/images/step5.png"/>
+For the question **Which API are you using?**, select "Google Cloud Messaging".
```

Comment:
Again, this is not what I see. I have a popup titled "APIs Credentials" with a pulldown button where I have to choose between API key, OAuth client Id, and Service account key.